### PR TITLE
Support luxon 3's options argument in fromObject

### PR DESCRIPTION
**Pull request by kensnyder**
Also:
Fix some typos in the README
Update tests to test luxon versions 1, 2, and 3

Thanks to @senning for the code changes and unit test in #6!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # luxon-parser
 
-[![NPM Link](https://img.shields.io/npm/v/luxon-parser?v=0.9.3)](https://npmjs.com/package/luxon-parser)
-[![Build Status](https://ci.appveyor.com/api/projects/status/github/kensnyder/luxon-parser?branch=master&svg=true&v=0.9.3)](https://ci.appveyor.com/project/kensnyder/luxon-parser/branch/master)
-[![Code Coverage](https://codecov.io/gh/kensnyder/luxon-parser/branch/master/graph/badge.svg?v=0.9.3)](https://codecov.io/gh/kensnyder/luxon-parser)
-[![ISC License](https://img.shields.io/npm/l/luxon-parser.svg?v=0.9.3)](https://opensource.org/licenses/ISC)
+[![NPM Link](https://img.shields.io/npm/v/luxon-parser?v=0.9.4)](https://npmjs.com/package/luxon-parser)
+[![Build Status](https://ci.appveyor.com/api/projects/status/github/kensnyder/luxon-parser?branch=master&svg=true&v=0.9.4)](https://ci.appveyor.com/project/kensnyder/luxon-parser/branch/master)
+[![Code Coverage](https://codecov.io/gh/kensnyder/luxon-parser/branch/master/graph/badge.svg?v=0.9.4)](https://codecov.io/gh/kensnyder/luxon-parser)
+[![ISC License](https://img.shields.io/npm/l/luxon-parser.svg?v=0.9.4)](https://opensource.org/licenses/ISC)
```
Comment (kensnyder) [inline README.md:6]: Bumped versions in the readme.

File: README.md [hunk 2]
```diff
@@ -95,14 +95,14 @@ from\*() function:
 - Twitter
 
 `luxon-parser` relies on
-[any-date-format](https://www.npmjs.com/package/any-date-parser) which supports
+[any-date-parser](https://www.npmjs.com/package/any-date-parser) which supports
```
Comment (kensnyder) [inline README.md:98]: A typo here and a few below.

File: index.js [hunk 3]
```diff
@@ -49,7 +49,7 @@ DateTime.fromAny = function fromAny(date, options = {}) {
 	} else if (typeof date === 'number') {
 		return DateTime.fromMillis(date, options);
 	} else if (typeof date === 'object') {
-		return DateTime.fromObject(date);
+		return DateTime.fromObject(date, options);
```
Comment (kensnyder) [inline index.js:52]: @senning's change

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "luxon-parser",
-	"version": "0.9.3",
+	"version": "0.9.4",
```
Comment (kensnyder) [inline package.json:3]: bump for npm release.

File: package.json [hunk 5]
```diff
@@ -35,11 +35,11 @@
 		"any-date-parser": "1.5.3"
 	},
 	"peerDependencies": {
-		"luxon": "^1.26.0"
+		"luxon": ">=1.28.0"
```
Comment (kensnyder) [inline package.json:38]: support luxon 1 through 3 since as of today, luxon is still making releases to all three versions.

File: package.json [hunk 6]
```diff
@@ -35,11 +35,11 @@
...
 	},
 	"devDependencies": {
-		"eslint": "8.7.0",
-		"jest": "27.4.7",
-		"prettier": "2.5.1"
+		"eslint": "8.32.0",
+		"jest": "29.3.1",
+		"prettier": "2.8.3"
```
Comment (kensnyder) [inline package.json:43]: update old dev dependencies

File: test/luxon-parser.spec.js [hunk 7]
```diff
@@ -83,6 +83,28 @@ describe('DateTime.fromAny()', () => {
...
+		const optionsFaroe = {
+			zone: 'Atlantic/Faroe',
+		};
+		const actualFiji = DateTime.fromAny(object, optionsFiji);
+		const expectedFiji = DateTime.fromObject(object, optionsFiji);
+		expect(actualFiji.toString()).toBe(expectedFiji.toString());
+		const actualFaroe = DateTime.fromAny(object, optionsFaroe);
+		const expectedFaroe = DateTime.fromObject(object, optionsFaroe);
+		expect(actualFaroe.toString()).toBe(expectedFaroe.toString());
+	});
```
Comment (kensnyder) [inline test/luxon-parser.spec.js:107]: @senning's test

File: test/run-jest.sh [hunk 8]
```diff
@@ -22,19 +22,32 @@ fi
...
 then
-  # make sure luxon is available
-  if ! npm list -g | grep -q 'luxon'
+  # check if luxon is installed globally
+  if [[ -z "$(npm list -g | grep -q 'luxon')" ]]
   then
-    echo "${GREEN}Attempting to globally install luxon with npm for unit tests...${WHITE}"
-    echo "npm install -g luxon@1"
-    npm install -g luxon@1
+    for version in {1..3}
```
Comment (kensnyder) [inline test/run-jest.sh:28]: Test luxon version 1, 2 and 3 in a loop.

**Review by senning — APPROVED** [review]
This looks great! Thanks for the quick update!

**Pull request merged**